x-pack/plugins/alerting/server/usage/lib/get_telemetry_from_kibana.ts

```diff
@@ -490,3 +506,58 @@ export async function getTotalCountInUse({
     };
   }
 }
+
+export async function getMWTelemetry({
+  savedObjectsClient,
+  logger,
+}: MWOpts): Promise<GetMWTelemetryResults> {
+  try {
+    const mwFinder = savedObjectsClient.createPointInTimeFinder<MaintenanceWindowAttributes>({
+      type: MAINTENANCE_WINDOW_SAVED_OBJECT_TYPE,
+      namespaces: ['*'],
+      perPage: 100,
```

Comment:
nit: What do you think of using the `field` parameter to reduce the response size to only the fields we want? I think the `events` attribute can be quite big.